tests/Fixtures/Integration/misc/PHP7_3.test

```diff
@@ -98,6 +102,7 @@ final class MyTest extends \PHPUnit_Framework_TestCase
 {
     public function testFoo(): void
     {
+        $this->setExpectedException('RuntimeException', 'Msg', 123); // `php_unit_expectation` rule
```

Comment:
can you add a trailing ',' after `123` here?

Reply:
Of course :man_facepalming: - done.